### PR TITLE
Update badges

### PR DESCRIPTION
**Pull request by mmitche**
/cc @iamjasonp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
+|**Release**|[![Build and Innerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/windows_release.svg?label=build+%26+innerloop+test)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/windows_release/) [![Outerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/outerloop_windows_release.svg?label=outerloop+tests)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/job/outerloop_windows_release/)|[![Build and Innerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/linux_release.svg?label=build+%26+innerloop+test)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/job/linux_release/)|
```
Comment (iamjasonp) [inline README.md]: Link for Release/Windows/build+innerloop incorrect (should be /job/master/job/windows_release)

**Comment by iamjasonp** [comment]
Oddly, I'm finding that when viewing the new badges in preview, I see all of them as "building" rather than as their actual state (some green - "passing", some red - "failing", some "not built"). Any thoughts on why this is happening? It could be one of those github asset caching oddities that is causing this, and maybe it'll be correct on merge.

**Comment by iamjasonp** [comment]
[https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/)<build>.svg?label=build+%26+innerloop+test seems to be providing the correct output 

(missing extra /job/ after "dotnet_wcf" and before "master"
[https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/)<build>.svg?label=build+%26+innerloop+test

**Comment by iamjasonp** [comment]
[http://www.example.com](http://www.example.com)

**Comment by iamjasonp** [comment]
[test](http://www.example.com)

**Comment by mmitche** [comment]
@iamjasonp Thanks for the heads up.  Odd that there is no /job/ before the last element.  Fixed.

**Comment by iamjasonp** [comment]
LGTM. Thanks for doing this!

**Pull request merged**